### PR TITLE
bugfix: check app organization when CheckUserPermission

### PR DESCRIPTION
**Pull request by dplynskiy**

### CODE REVIEW

File: object/check.go [hunk 1]
```diff
@@ -483,7 +483,17 @@ func CheckUserPermission(requestUserId, userId string, strict bool, lang string)
 
 	hasPermission := false
 	if strings.HasPrefix(requestUserId, "app/") {
-		hasPermission = true
+		requestApp, err := GetApplication(ctx, fmt.Sprintf("admin/%s", strings.Split(requestUserId, "/")[1]))
+		if err != nil {
+			return false, err
+		}
+		if requestApp == nil {
+			return false, fmt.Errorf(i18n.Translate(lang, "check:Session outdated, please login again"))
```
Comment (ISulimanov) [inline object/check.go:491]: mb application not found?

File: object/check.go [hunk 2]
```diff
@@ -457,7 +457,7 @@ func CheckPassErrorToMessage(err error, lang string) string {
 	return err.Error()
 }
 
-func CheckUserPermission(requestUserId, userId string, strict bool, lang string) (bool, error) {
+func CheckUserPermission(ctx context.Context, requestUserId, userId string, strict bool, lang string) (bool, error) {
```
Comment (ISulimanov) [inline object/check.go:460]: what about global admin?

File: object/check.go (hunk 1 above)
Comment (dplynskiy) [reply object/check.go:491]: it's copied from block for user for sameness of error in func. It's means that you use session for app that was deleted or renamed

File: object/check.go (hunk 2 above)
Comment (dplynskiy) [reply object/check.go:462]: fixed

**Review by ISulimanov — APPROVED** [review]

**Pull request merged**